docs/support/support_overview.md

```diff
@@ -0,0 +1,91 @@
+<!--
+  ~ Copyright (c) 2025 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# Arista AVD A-Care TAC Support Overview
+
+Arista AVD is a multi-domain network automation framework consumable as code, providing a continuous design framework. The AVD project is offered as an open-source project and backed by the world-class Arista Technical Support team through A-Care Service. TAC support for AVD must be purchased separately; see the ordering information below.
+
+## Support Offering
+
+- AVD software is covered by [A-Care Priority Levels](https://www.arista.com/assets/data/tac/downloads/SRPriorityLevels.pdf).
+- TAC support covers software defects and troubleshooting Q&A.
```

Comment:
defects -> bugs

Reply:
Let's stay with defects:

- In formal software engineering, quality assurance, and technical documentation, "defect" is generally considered the more precise and professional term.
- In everyday communication among developers or when speaking to a less technical audience, "bug" is perfectly acceptable and often more readily understood.